CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_twilio v0.4.0
+
+## Features
+- Added the ability to disable the `USAGE_RECORD` source table and its downstream fields via variable `using_twilio_usage_record` (default `true`). Refer to the [README](https://github.com/fivetran/dbt_twilio?tab=readme-ov-file#step-4-enablingdisabling-models) for more details.
+
+## Under the hood
```

Comment:
```suggestion
## Under the Hood
```
I think we've always been capitalizing nouns, so just want to keep it consistent.

Reply:
Thanks!